jOOQ/src/main/java/org/jooq/GrantGrantedStep.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Other licenses:
+ * -----------------------------------------------------------------------------
+ * Commercial licenses for this work are available. These replace the above
+ * ASL 2.0 and offer limited warranties, support, maintenance, and commercial
+ * database integrations.
+ *
+ * For more information, please visit: http://www.jooq.org/licenses
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ *
+ */
+package org.jooq;
+
+import static org.jooq.SQLDialect.FIREBIRD;
+import static org.jooq.SQLDialect.HSQLDB;
+import static org.jooq.SQLDialect.MYSQL;
+import static org.jooq.SQLDialect.POSTGRES;
+
+/**
+ * The step of creation <code>GRANT</code> statement that assign user
+ * or role a grant option.
+ *
+ * @author Timur Shaidullin
+ */
+public interface GrantGrantedStep extends GrantFinalStep{
+    @Support({ POSTGRES, MYSQL, HSQLDB, FIREBIRD })
```

Comment:
Just to comment, I'll remove the individual dialect references until we cover the functionality in our integration tests. While it is quite probable that this is correct, we want to *formally* support the feature by guaranteeing that it works...

Reply:
Do you plan make an integration test open in the future? This step a bit decrease your workload, if users write integration test by themselves.

Reply:
No, the integration tests are closed source and available to our paying customers only. We follow a similar dual-licensing business model as SQLite, which also does not open source all the tests.